app/controllers/rating_questions_controller.rb

```diff
@@ -1,5 +1,70 @@
+# require 'pry'
+
 class RatingQuestionsController < ApplicationController
+  before_action :set_rating_question, only: [:show, :update, :destroy]
+
   def index
     @rating_questions = RatingQuestion.all
   end
+
+  def new
+    @rating_question = RatingQuestion.new
+  end
+  
+  def question_params
+    params.require(:rating_question).permit(:title, :tag)
+  end
+
+  def create
+   if request.body.size.zero?
+    return render json: {}, status: 400
+   end
+    @rating_question = RatingQuestion.create(question_params)
+    if @rating_question.save
+      respond_to do |format|
+        format.html { redirect_to "/rating_questions", notice: "Your question has been created." }
+        format.json { render :show, status: 201 }
+      end
+    else
+      respond_to do |format|
+        errors = {"errors" => @rating_question.errors.messages}
+        format.html { redirect_to "/rating_questions/new", notice: "Must have a title" }
+        format.json { render json: errors, status: 422 }
+      end
+    end
+  end
+
+  def destroy
+    @rating_question = RatingQuestion.find(params[:id])
+    @rating_question.destroy
+    redirect_to "/rating_questions"
```

Comment:
This should not `redirect_to "/rating_questions"`, as a JSON request should receive a 204 status. Likely you want to use a `respond_to` here as well:

```ruby
respond_to do |format|
  format.html { redirect_to "/rating_questions" }
  format.json { head 204 }
end
```

Reply:
yeah i was trying that but im getting this error 
Failure/Error: expect(response.status).to eq(204)

       expected: 204
            got: 302